nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -452,7 +452,7 @@ in
     system.activationScripts.nix = stringAfter [ "etc" "users" ]
       ''
         # Create directories in /nix.
-        ${nix}/bin/nix ping-store --no-net
+        ${nix}/bin/nix --option experimental-features nix-command ping-store --no-net
```

Comment:
Probably we should replace `nix ping-store` by some other dummy command that opens the store. I think even something like `nix-store -q` without arguments will do this.